content/docs/2.0/scalers/redis-lists.md

```diff
@@ -9,7 +9,7 @@ go_file = "redis_scaler"
 
 ### Trigger Specification
 
-This specification describes the `redis` trigger that scales based on the length of a list in Redis.
+This specification describes the `redis` trigger that scales based on the length of a list,hash,set or sorted-set in Redis.
 
 ```yaml
```

Comment:
Not sure if `listName` & `listLength` are the ideal names for this

Reply:
IMHO we can keep the old ones, If we don't come up with better names. (I don't have a better idea 🤷‍♂️ 😄 )